examples/min-platform/embedding/src/wasi.rs

```diff
@@ -301,7 +301,7 @@ impl InputStream for NeverReadable {
     }
 }
 
-// WriteLog is used implement stdout and stderr. Clonable because wasi:cli
+// WriteLog is used implement stdout and stderr. Cloneable because wasi:cli
```

Comment:
FWIW, https://github.com/rust-lang/rust actually uses both, but *cloneable* seems to be more common:
- https://github.com/search?q=repo%3Arust-lang%2Frust+cloneable&type=code
- https://github.com/search?q=repo%3Arust-lang%2Frust+clonable&type=code